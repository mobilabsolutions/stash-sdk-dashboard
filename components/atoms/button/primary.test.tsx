/* eslint-env jest */

import React from 'react'
import renderer from 'react-test-renderer'

import { withTestSetup } from '../../../test_utils'
import PrimaryButton from './primary'

const TestButton = withTestSetup(PrimaryButton)

it('PrimaryButton Should Render', () => {
  const component = renderer.create(<TestButton label="Login" />)
  const tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})

it('Disabled PrimaryButton Should Render', () => {
  const component = renderer.create(<TestButton label="Disabled" disabled />)
  const tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
