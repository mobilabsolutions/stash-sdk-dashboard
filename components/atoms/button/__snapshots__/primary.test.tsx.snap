// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Disabled PrimaryButton Should Render 1`] = `
<button
  className="primary__Button-sc-1lwi4g7-0 kUxHGQ"
  disabled={true}
  onClick={null}
  type="submit"
>
  <span>
    Disabled
  </span>
</button>
`;

exports[`PrimaryButton Should Render 1`] = `
<button
  className="primary__Button-sc-1lwi4g7-0 kUxHGQ"
  disabled={false}
  onClick={null}
  type="submit"
>
  <span>
    Login
  </span>
</button>
`;
